src/components/forms/Label/Label.tsx

```diff
@@ -7,14 +7,18 @@ interface LabelProps {
   className?: string
   error?: boolean
   hint?: React.ReactNode
+  search?: boolean
 }
 
 export const Label = (props: LabelProps): React.ReactElement => {
-  const { children, htmlFor, className, error, hint } = props
+  const { children, htmlFor, className, error, hint, search } = props
 
   const classes = classnames(
-    'usa-label',
-    { 'usa-label--error': error },
+    {
+      'usa-label': !search,
+      'usa-sr-only': search,
```

Comment:
`usa-sr-only` refers to "screen reader only", as in the text is not visible to the user but is still present so it can be read if using a screen reader - it's not exclusive to the search form. So I think the prop should be `srOnly`